roqoqo/src/devices/mod.rs

```diff
@@ -220,7 +220,7 @@ pub trait Device {
     #[allow(unused_mut)]
     fn change_device(&mut self, hqslang: &str, operation: &[u8]) -> Result<(), RoqoqoBackendError> {
         Err(RoqoqoBackendError::GenericError {
-            msg: "The device ".to_string(),
+            msg: "The `change_device()` method has not been implemented.".to_string(),
```

Comment:
There doesn't seem to be a corresponding unittest for this, could you add this? Also, the docstring has not been updated (neither has the pyo3 docstring) to reflect the change in error message

Reply:
I forgot about the tests, now the current devices are tested.
I explained a bit better the error in the doc.
What pyo3 docstring do you mean exactly?

Reply:
If this function exists in the pyo3 interface (probably in the macros here?), has the `Raises` section been updated?

Reply:
This docstring and the unittests look great